README.md

```diff
@@ -1,2 +1,7 @@
-# photos-transfer
-Script that copy photos from SD card to directory with date for name.
+photos-transfer
+=======
+Script that copies photos from an SD card to a directory with the date as the name.
+
+License
+=======
+[here!](https://github.com/sablepear/photos-transfer/blob/master/LICENSE)
```

Comment:
Can you change this by `https://github.com/Kevinrob/photos-transfer/blob/master/LICENSE`?